### DISCUSSION TITLE
关于cache插件的疑问

### DISCUSSION THREAD

**Discussion by wangmice** [IrineSistiana/mosdns#451] [General]
cache插件命中后会中止后续插件的运行，但是假如我想把某个域名ip放入ipset或nftset，只要是cache命中这个域名，是不是也不会执行后续的ipset和nftset插件？有什么办法解决吗？

**Comment by BozhanL** [comment]
cache 的这个不知道行不行
```
# 当缓存命中后，需要执行的插件。
when_hit: ""
```

**Reply by BozhanL** [reply]
https://irine-sistiana.gitbook.io/mosdns-wiki/mosdns/cha-jian-ji-qi-can-shu#cache-huan-cun

**Reply by wangmice** [reply]
没注意到有这个，貌似可以，我试试

**Comment by BozhanL** [comment]
还有一种方法 [用ttl插件把ttl限制的比ipset的timeout小](https://github.com/IrineSistiana/mosdns/issues/339#issuecomment-1117216086)
